### PR TITLE
Add school pages for 5495

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to [688](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/688)

Adds the school selection pages and updates the 1995 to use a shared definition for one of them.

screen shot 2017-04-12 at 2 58 48 pm
screen shot 2017-04-12 at 2 58 58 pm
screen shot 2017-04-12 at 2 59 02 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
There are a bunch of failing tests for 5490, but I'm not sure exactly why.

**Comment by cvalarida** [comment]
I assume it's right, but we only want the address shown conditionally on the first page, right?

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1821,7 +1821,7 @@
         "babel-code-frame": {
           "version": "6.22.0",
           "from": "babel-code-frame@>=6.22.0 <7.0.0",
-          "resolved": "https://registry.npmjs.org/babel-code-frame/-/babel-code-frame-6.22.0.tgz"
+          "resolved": "http://registry.npmjs.org/babel-code-frame/-/babel-code-frame-6.22.0.tgz"
```
Comment (jbalboni) [inline npm-shrinkwrap.json]: Beats me why this is happening. But this was my third and most successful attempt at updating the shrinkwrap file.

Comment (cvalarida) [inline npm-shrinkwrap.json]: Eeee....http makes me uncomfortable. 0.o

File: test/edu-benefits/5495/config/schoolSelection.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/5495/config/form.js';
+
+describe('Edu 5495 schoolSelection', () => {
```
Comment (cvalarida) [inline test/edu-benefits/5495/config/schoolSelection.unit.spec.jsx]: Is this just to test that it works with the 5495 schema? (What with it being a shared page and all.)

**Review by cvalarida — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -166,7 +166,7 @@
     "tinyliquid": "^0.2.33",
     "url-loader": "^0.5.7",
     "uswds": "^1.0.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#7ef76dfb7e334a23db5b663a8c0dff15747748f6",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema#32580687bb7308e638a1d6ef675ca7a92cc4cd35",
```
Comment (cvalarida) [inline package.json]: I think this has to do with why the 5490 tests are breaking. Check out [the diff](https://github.com/department-of-veterans-affairs/vets-json-schema/compare/7ef76dfb7e334a23db5b663a8c0dff15747748f6...32580687bb7308e638a1d6ef675ca7a92cc4cd35). The broken tests are all in `benefitHistory` which is what's modified between these two commits. (That's probably the problem with the schema test as well...)

Not that I know the _actual_ cause, mind you, but I'd look into the diff between the two commits in vets-json-schema.

**Review by cvalarida — APPROVED** [review]
Looks good. Just a suggestion on where to look for the failing tests.

File: test/edu-benefits/5495/config/schoolSelection.unit.spec.jsx (hunk 2 above)
Comment (jbalboni) [inline test/edu-benefits/5495/config/schoolSelection.unit.spec.jsx]: Yeah. It's kind of debatable.

File: package.json (hunk 3 above)
Comment (jbalboni) [inline package.json]: I have it figured out, but I have to make another schema change, so this won't pass quite yet.

File: npm-shrinkwrap.json (hunk 1 above)
Comment (jbalboni) [inline npm-shrinkwrap.json]: I'll change it back by hand.

File: src/js/edu-benefits/5490/config/form.js [hunk 4]
```diff
@@ -254,8 +256,8 @@ const formConfig = {
                 'transferOfEntitlement',
                 'veteranFullName',
                 'view:veteranId',
-                'other',
-                'view:noPreviousBenefits'
+                'view:noPreviousBenefits',
```
Comment (raquelromano-gov) [inline src/js/edu-benefits/5490/config/form.js]: Are you sure? Last I heard, this was supposed to go back to the top (last comment on department-of-veterans-affairs/vets.gov-team#1808)

Comment (jbalboni) [inline src/js/edu-benefits/5490/config/form.js]: You're right. I confirmed with Alex the other day, just hadn't made the change yet.

**Pull request merged**